src/pages/Events.tsx

```
@@ -72,122 +104,160 @@ const Events: React.FC = () => {
-    switch (eventType) {
+  if (events.length === 0) {
+    return (
+      <EmptyState
+        title="No events yet"
+        description="Events from your watched repositories will appear here"
+        icon={<GitPullRequest className="h-12 w-12" />}
+      />
+    );
+  }
+
+  const getEventIcon = (event: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)